app/routes/docs/history.mdx

```diff
@@ -0,0 +1,48 @@
+
+## BACODINE
+
+Dr. Scott Barthelmy at NASA/GSFC realized that this new data mode created an opportunity -- near-time notification of gamma-ray bursts detected by CGRO's Burst And Transient Search Experiment ([BATSE](https://heasarc.gsfc.nasa.gov/docs/cgro/batse/)) instrument. He went on to launch the BAtse COordinates DIstribution NEtwork (BACODINE) system in 1993. Software monitored the real-time telemetry of BATSE and produced an automatic localization within seconds, with a later improvement of a more accurate localization produced by BATSE scientists, after a delay of tens of minutes. The BATSE scientist were notified by BACODINE via pager when a transient occurred. They localized the source and sent the information to the BACODINE system for distribution.
+
+The [GCN Notices](/notices) distribution mechanisms expanded over the following years: phone connections (July 1993), e-mail (December 1993), direct computer-to-computer socket connections over the Internet (January 1994), and by alpha-numeric pagers (September 1994) to any interested follow-up observatories within seconds of detection.
+
+The first robotic follow-up instrument to connect to BACODINE was the Gamma-Ray Optical Counterpart Search Experiments ([GROCSE](https://ui.adsabs.harvard.edu/abs/1995Ap%26SS.231..255A/abstract)) in August 1993. Additional capabilities were added to BACODINE throughout the mid 1990's including trigger classification (GRBs from non-GRBs), monitoring (including 24/7 staffing) and the modern incarnation of the interplanetary network ([IPN](https://heasarc.gsfc.nasa.gov/docs/heasarc/missions/ipn.html)).
+
+## Gamma-ray Burst Coordinates Network
+
+New instruments were added to the network, including the Rossi X-ray timing Explorer ([RXTE](https://heasarc.gsfc.nasa.gov/docs/xte/learning_center/index.html)) Proportional Counter Array ([PCA](https://heasarc.gsfc.nasa.gov/docs/xte/learning_center/pca.html)) and the Array of Low Energy X-Ray Imaging Sensors ([ALEXIS](https://heasarc.gsfc.nasa.gov/docs/heasarc/missions/alexis.html)) extreme ultraviolet transient mission. GCN Notices from the CGRO Imaging Compton Telescope ([COMPTEL](https://heasarc.gsfc.nasa.gov/docs/cgro/comptel/)) were added in November 1997. In recognition of the broader usage, the system was renamed the Gamma-ray burst Coordinates Network (GCN) in 1997.
```

Comment:
Review these links. Some of these look like public outreach pages. For example, how about this for XTE? https://heasarc.gsfc.nasa.gov/docs/xte/XTE.html